src/category_theory/adjunction/basic.lean

```diff
@@ -204,16 +204,87 @@ def id : 𝟭 C ⊣ 𝟭 C :=
   unit := 𝟙 _,
   counit := 𝟙 _ }
 
+/-- If F and G are naturally isomorphic functors, establish an equivalence of hom-sets. -/
+def equiv_homset_left_of_nat_iso
+  {F G : C ⥤ D} (iso : F ≅ G) {X : C} {Y : D} :
```

Comment:
Knowing what's coming, I would probably use variable names `{F F' : C ⥤ D}` here, and `{G G' : D ⥤ C}` below.